### DISCUSSION TITLE
Looking for help to create a simple mobile app (ExpressNoctik)

### DISCUSSION THREAD

**Discussion by Venividivici99** [community/community#147355] [Actions]
### Select Topic Area

General

### Body

Hello everyone,
I am working on a project called ExpressNoctik, a mobile app designed to provide nighttime delivery services for essential products (bread, medicine, water, etc.) in my locality. I am looking for people willing to help me develop this app.

My needs are:
Creation of a simple, intuitive, and functional app.

My needs are:
Creation of a simple, intuitive, and functional app.
Assistance in choosing between a No-Code approach or custom development.
Guidance on the technical setup and deployment.

I currently lack financial resources to pay for the development, but I am open to collaborating with volunteers or developers passionate about social and innovative projects. Any help, even advice, would be greatly appreciated.

Thank you in advance for your time and support!
Feel free to reach out if you’re interested or have any questions.

**Comment by Logophoman** [comment]
Your project sounds nice but also quite ambitious to be honest, especially if you're not into coding and lacking funding. 

### 1. **No-Code vs. Custom Development**
If you want something quick and budget-friendly, try a No-Code platform like **Adalo, Glide, or Bubble**. They let you build functional apps without coding. But if you need more control (like real-time tracking or custom features), custom development with **React Native (for the app)** and **Node.js/Firebase (for the backend)** will probably be the way to go. No-Code solutions will usually and most likely make you hit a brick wall in terms of functionality 🫤

### 2. **Technical Setup**
So then for a custom build, consider:
- **Frontend**: React Native (cross-platform app)
- **Backend**: Supabase (authentication, real-time DB) or Node.js/Express
- **Database**: Supabase with PostgreSQL (for simplicity) or plain PostgreSQL
- **Hosting**: Vercel (easy but not the cheapest) or self hosted with Apache2 or Nginx...

If this already sounds complex, start with Supabase and Vercel as they handle most of the complicated things for you...

### 3. **Finding Help**
Since you’re looking for volunteers, try:
- **GitHub Discussions/Issues** (well you're here)
- **Reddit** (r/ProgrammingBuddies or so)
- **Dev.to** or hackathons on **DevPost**
- Platforms like **Catchafire.org** for volunteers

### 4. **General Advice**
If you try to code youself start small with basic features like auth and then go on and do something like placing an order and then add more features later. You can get Help From ChatGPT or so for coding... If you document your idea clearly, it’s easier to get help from developers and also here...

Best of luck with your project, and I hope you’ll find people willing to support it!

BTW what region is it for? Often local help is best 😉

**Comment by Venividivici99** [comment]
*Hello,*

Thank you so much for your detailed response and advice.

I want to clarify that I have no experience in app development, which makes
this process quite intimidating for me. I’ve already tried using No-Code
platforms like Glide or Bubble, but I failed because I was working on my
Android phone, which is very limited in terms of features and ease of use.

That said, your suggestion about using React Native and Supabase for a
custom build interests me greatly, especially if experienced volunteers can
help me move this project forward.

I also want to emphasize that my app will include inclusive initiatives
such as SOS Leïla, a nighttime support program for emergencies (e.g.,
medication, essential food items), and School Food Credit, aimed at
providing basic food supplies to children from struggling families. I am
confident this can have a significant positive impact in my community.

To answer your question, my project is intended for the western region of
Côte d’Ivoire in West Africa, where nighttime delivery services for
essential goods are almost nonexistent. With the right support, I believe
this idea can genuinely address an urgent need in the area.

If you have other recommendations, contacts, or suggestions to help me
overcome these challenges, I would truly appreciate it. In the meantime, I
will explore the other platforms you mentioned (Reddit, DevPost, etc.) to
seek support.


Thank you again for your help!


Sincerely,

Abdoul Coulibaly

Le mer. 18 déc. 2024 à 11:15, Logophoman ***@***.***> a
écrit :
